coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -144,6 +144,21 @@ def convert_block(context, block, inputs):
     14: torch.qint32,
 }
 
+TORCH_DTYPE_TO_NUM = {
```

Comment:
Is it possible to use the existing `NUM_TO_TORCH_DTYPE` to construct this `TORCH_DTYPE_TO_NUM`, by something like `{torch_dtype: num for num, torch_dtype in NUM_TO_TORCH_DTYPE.items()}`?